bedrock/mozorg/templates/mozorg/about/governance/organizations.html

```diff
@@ -20,7 +20,7 @@ <h1 class="title-shadow-box">{{_('Mozilla Organizations')}} </h1>
 
 <p>{{_('The <a href="//www.mozilla.org/foundation/">Mozilla Foundation</a> is a California non-profit corporation exempt from Federal income taxation under IRC 501(c)(3). The Foundation supports the existing Mozilla community and oversees Mozilla’s governance structure. It also actively seeks out new ways for people around the world to recognize and steward the Internet as a critical public resource.')}}</p>
```

Comment:
Actually, it would be even better to use the `url` helper here for the mozorg URLs. So for example here:

    <p>
        {% trans foundation_url=url('foundation.index') %}
	    The <a href="{{ foundation_url }}">Mozilla Foundation</a> is a California non-profit corporation exempt from Federal income taxation under IRC 501(c)(3). The Foundation supports the existing Mozilla community and oversees Mozilla’s governance structure. It also actively seeks out new ways for people around the world to recognize and steward the Internet as a critical public resource.
        {% endtrans %}
    </p>

This will then use the appropriate URL automatically and take protocol without needing to think about it.